Sources/AccessibilitySnapshot/iOSSnapshotTestCase/ObjC/include/FBSnapshotTestCase_Accessibility.h

```diff
@@ -14,47 +14,47 @@
 //  limitations under the License.
 //
 
-#define SnapshotVerifyAccessibility(view__, identifier__)\
+#define SnapshotVerifyAccessibility(view__, identifier__, perPixelTolerance__)\
```

Comment:
We should keep the basic two argument macro and add a new one for tolerance. I think many consumers will still want to non-tolerance version.